### ISSUE TITLE
Prior bulk zip files are deleted on download

### ISSUE THREAD

**Issue by nesnoj** [OpenEnergyPlatform/open-MaStR#564]
Alle files in the xml dir [are cleared](https://github.com/OpenEnergyPlatform/open-MaStR/blob/e03e824885d4f421c30aeb7c97f59f2570bea9ef/open_mastr/xml_download/utils_download_bulk.py#L112-L113) when a download is triggered. This is annoying when users want to keep old releases.
The xml dir needs to be created though, but this is redundantly done [before the download starts](https://github.com/OpenEnergyPlatform/open-MaStR/blob/e03e824885d4f421c30aeb7c97f59f2570bea9ef/open_mastr/mastr.py#L228).

Is there a reason why it is done this way - any objections against removing this behaviour? @FlorianK13

**Comment by FlorianK13** [comment]
Yes, we did not want to fill up storage from users who are not aware that every download is saved forever. I would argue for keeping this as default behavior, but we can add a parameter `keep_old_downloads` to keep the existing zipped files?

**Comment by nesnoj** [comment]
That's reasonable! However, a quiet deletion is annoying and a confirmation was helpful.
And a param for keeping old versions sounds good to me.
Thx for the quick feedback!
